app/models/tax_tribunal_decision.rb

```diff
@@ -0,0 +1,9 @@
+require "document_metadata_decorator"
+
+class TaxTribunalDecision < DocumentMetadataDecorator
+  set_extra_field_names [
+    :hidden_indexable_content,
+    :tribunal_decision_category,
+    :tribunal_decision_decision_date
```

Comment:
Can you add a trailing comma here in accordance with the [styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#collections)?